--- bullet3/CMakeLists.txt	2025-09-03 20:13:03.357562300 +0200
+++ bullet3/CMakeLists.txt.new	2025-09-03 22:40:37.625700000 +0200
@@ -1,10 +1,29 @@
 cmake_minimum_required(VERSION 3.5)
 set(CMAKE_ALLOW_LOOSE_LOOP_CONSTRUCTS true)
+add_subdirectory(bullet3)
+set(OPENCL_FOUND TRUE)
+set(OPENCL_INCLUDE_DIR "C:/OpenCL-SDK/include/CL")
+set(OPENCL_LIBRARY "C:/OpenCL-SDK/lib/OpenCL.lib")
+set(USE_DOUBLE_PRECISION OFF CACHE BOOL "Use double precision" FORCE)
+set(USE_OPENCL ON CACHE BOOL "" FORCE)
+set(BUILD_BULLET3 ON CACHE BOOL "" FORCE)
+set(BT_USE_PROFILE ON CACHE BOOL "Enable Bullet profiling" FORCE)
+
+PROJECT(BULLET_PHYSICS LANGUAGES CXX)
+find_package(OpenMP REQUIRED)
+add_definitions(-DBT_USE_OPENMP=1 -DBT_THREADSAFE=1)
+set(OPENMP_FOUND TRUE)
+set(USE_OPENMP ON CACHE BOOL "Enable OpenMP parallelization" FORCE)
+set(BULLET2_MULTITHREADING ON CACHE BOOL "Enable OpenMP parallelization" FORCE)
+set(BULLET2_USE_OPEN_MP_MULTITHREADING ON CACHE BOOL "Enable OpenMP parallelization" FORCE)
+set(BT_USE_OPENMP ON CACHE BOOL "Enable OpenMP support" FORCE)
+set(BT_THREADSAFE ON CACHE BOOL "Enable thread safety in Bullet" FORCE)
+
 cmake_policy(SET CMP0017 NEW)
 #this line has to appear before 'PROJECT' in order to be able to disable incremental linking
 SET(MSVC_INCREMENTAL_DEFAULT ON)
 
-PROJECT(BULLET_PHYSICS)
+
 FILE (STRINGS "VERSION" BULLET_VERSION)
 
 IF(COMMAND cmake_policy)
@@ -32,9 +51,9 @@
 
 OPTION(ENABLE_VHACD "Use VHACD in BulletRobotics and pybullet" ON)
 
-OPTION(BULLET2_MULTITHREADING "Build Bullet 2 libraries with mutex locking around certain operations (required for multi-threading)" OFF)
+OPTION(BULLET2_MULTITHREADING "Build Bullet 2 libraries with mutex locking around certain operations (required for multi-threading)" ON)
 IF (BULLET2_MULTITHREADING)
-    OPTION(BULLET2_USE_OPEN_MP_MULTITHREADING "Build Bullet 2 with support for multi-threading with OpenMP (requires a compiler with OpenMP support)" OFF)
+    OPTION(BULLET2_USE_OPEN_MP_MULTITHREADING "Build Bullet 2 with support for multi-threading with OpenMP (requires a compiler with OpenMP support)" ON)
     OPTION(BULLET2_USE_TBB_MULTITHREADING "Build Bullet 2 with support for multi-threading with Intel Threading Building Blocks (requires the TBB library to be already installed)" OFF)
     IF (MSVC)
         OPTION(BULLET2_USE_PPL_MULTITHREADING "Build Bullet 2 with support for multi-threading with Microsoft Parallel Patterns Library (requires MSVC compiler)" OFF)
