--- bullet3/CMakeLists.txt	2025-09-01 14:39:13.000000000 +0000
+++ bullet3/CMakeLists.txt	2025-09-01 14:25:57.000000000 +0000
@@ -1,5 +1,16 @@
 cmake_minimum_required(VERSION 3.5)
 set(CMAKE_ALLOW_LOOSE_LOOP_CONSTRUCTS true)
+
+set(OPENCL_FOUND TRUE)
+set(OPENCL_INCLUDE_DIR "C:/OpenCL-SDK/include/CL")
+set(OPENCL_LIBRARY "C:/OpenCL-SDK/lib/OpenCL.lib")
+set(USE_DOUBLE_PRECISION OFF CACHE BOOL "Use double precision" FORCE)
+set(USE_OPENCL ON CACHE BOOL "" FORCE)
+set(BUILD_BULLET3 ON CACHE BOOL "" FORCE)
+set(BT_USE_PROFILE ON CACHE BOOL "Enable Bullet profiling" FORCE)
+set(USE_OPENMP ON CACHE BOOL "Enable OpenMP parallelization" FORCE)
+
+
 cmake_policy(SET CMP0017 NEW)
 #this line has to appear before 'PROJECT' in order to be able to disable incremental linking
 SET(MSVC_INCREMENTAL_DEFAULT ON)
